SIPS/sip-15.md

```diff
@@ -66,11 +94,15 @@ interface ILiquidations {
     // owner only
     function setLiquidationDelay(uint time) external;
 
-    function setLiquidationRatio(uint ratio) external;
+    function setLiquidationRatio(uint liquidationRatio) external;
 
-    function setLiquidationTargetRatio(uint target) external;
+    function setLiquidationPenalty(uint penalty) external;
```

Comment:
Update with `ISynthetix.liquidateDelinquentAccount(address account, uint susdAmount)`

Reply:
That is a good name

Reply:
yeah makes it clear its for a bad actor not just anyone